src/views/assignments.mustache

```diff
@@ -0,0 +1,67 @@
+{{> navbar}}
+
+<style>
+    table.dataTable tbody td {
+        vertical-align: middle;
+    }
+</style>
+
+<script>
+    $(document).ready(function() {
+        $('#table').DataTable( {
+            "ajax": {
+                "url": "/api/assignments",
+                "dataSrc": "data.assignments",
+                "type": "POST",
+                "_csrf": "{{csrf}}"
+            },
+            "paging":   true,
+            "lengthMenu": [[25, 50, 100, -1], [25, 50, 100, "All"]],
+            "columns": [
+                { "data": "instance_name" },
+                { "data": "device_uuid" },
+                { data: {
+                    _:    "time.formatted",
+                    sort: "time.timestamp"
+                } },
+                { "data": "enabled" },
```

Comment:
Maybe we could use a format here too. (Yes/No instead of 1/0)

Reply:
Absolutely